### PR TITLE
Update README-zh_CN.md

### PR DESCRIPTION
**Pull request by 140948940**
enableObjectSlots配置项中文文档缺少的一行注释

### 🤔 What is the nature of this change?

- [ ] New feature
- [ ] Fix bug
- [ ] Style optimization
- [ ] Code style optimization
- [ ] Performance optimization
- [ ] Build optimization
- [ ] Refactor code or style
- [ ] Test related
- [x] Other

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by agileago** [comment]
小伙，很机智哟

**Comment by agileago** [comment]
这个库这么长时间都没人更新了，不会弃坑了吧

**Comment by 140948940** [comment]
> 小伙，很机智哟

哈哈

File: packages/babel-plugin-jsx/README-zh_CN.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ const App = {
   },
 };
 
-// or
+// 或者，当`enableObjectSlots不是false时，您可以使用对象插槽
```
Comment (Amour1688) [inline packages/babel-plugin-jsx/README-zh_CN.md:285]:
```suggestion
// 或者，当 `enableObjectSlots` 不是 `false` 时，您可以使用对象插槽
```

**Pull request merged**